config/packages/security.yaml

```diff
@@ -52,6 +54,12 @@ security:
                 # The name of the route to redirect to after logging out
                 target: homepage
 
+            # needed because in tests we redefine the 'main' firewall to use
+            # HTTP Basic instead of he login form, so this firewall has
```

Comment:
there is  a typo in this sentence (`instead of he login form`)

Reply:
fixed